tests/test_monitor.py

```diff
@@ -94,20 +108,29 @@ def test_basic_monitor(monitor, tn_client, loop):
     assert 'Task' in resp
 
     resp = execute(tn, 'ps 123\n')
-    assert 'Task' in resp
+    assert 'TypeError' in resp
```

Comment:
Can you elaborate on this one? Why this change?

Reply:
Because the new code for looking up the method to execute will take the method (`do_ps` in this example) and give it any arguments supplied by the user (`123`). The TypeError is simply the error that Python raises when you give an incorrect amount of arguments to a function.

I am currently working on implementing more of the public interface of Cmd where this will picked up.